js/events/core/events_engine.js

```diff
@@ -595,6 +595,7 @@ initEvent(normalizeEventArguments(function(src, config) {
         that.originalEvent = src;
         that.type = src.type;
         that.currentTarget = undefined;
+        Object.prototype.hasOwnProperty.call(src, 'isTrusted') && (that.isTrusted = src.isTrusted);
```

Comment:
1. Make code readable
```suggestion
        if(Object.prototype.hasOwnProperty.call(src, 'isTrusted')) {
            that.isTrusted = src.isTrusted;
        }
```

2. Let's get rid of `that` variable in this function.

Reply:
done